docs/vc-http-api.yml

```diff
@@ -116,6 +138,29 @@ paths:
 
 components:
   schemas:
+    UpdateCredentialStatus:
+      type: object
+      description: Request for updating the status of an issued credential.
+      properties:
+        credentialId:
+          type: string
+        credentialStatus:
+          type: array
+          items:
+            type: object
+            properties:
+              type:
+                type: string
+              status:
+                type: number
+      example:
+        { 
+          "credentialId": "urn:uuid:45a44711-e457-4fa8-9b89-69fe0287c86a",
+          "credentialStatus": [{
+            "type": "RevocationList2020Status",
+            "status": 0
```

Comment:
feels like there should be an index in here....

updateCredentialStatus = (credentialId:string, statusIndex:number, statusFlag: boolean)

Reply:
@tplooker ^

Reply:
I would have assumed that the `credentialId` would allow you to look that up? Having to supply it is exposing too much complexity to the client IMO

Reply:
Requiring `status` to be a number feels very specific to RevocationList2020Status, no? Other credential status mechanisms in the future may have other concepts of status. Maybe allow status to be any string (and in the specific case of RevocationList2020Status the only valid string values are "0" and "1")?

Reply:
+1 @peacekeeper I have updated this

Reply:
@OR13 I have addressed @peacekeeper's query but will leave this open until you have had a chance to reply on your original query